src/options.rs

```diff
@@ -42,6 +42,9 @@ pub struct Options {
     /// Ignore a specified column.
     #[structopt(short = "I", number_of_values = 1)]
     pub ignore: Vec<String>,
+
+    #[structopt(short = "i", number_of_values = 1)]
```

Comment:
Let's provide a comment (it will be visible in `--help`) and update README to include a new option

Reply:
Done.